Packs/IntegrationsAndIncidentsHealthCheck/Scripts/GetFailedTasks/GetFailedTasks.yml

```diff
@@ -1,21 +1,32 @@
 args:
-- defaultValue: -status:closed
-  description: The query by which to retrieve failed tasks. Optional. The default value is "-status:closed"
+- default: false
+  defaultValue: -status:closed
+  description: The query by which to retrieve failed tasks. Optional. The default
+    value is "-status:closed"
+  isArray: false
   name: query
+  required: false
+  secret: false
+- default: false
+  description: The account name.
```

Comment:
maybe it should be tenant name, @altmannyarden WDYT?
@kirbles19 what's the usual term in the system?

Reply:
I talked with @altmannyarden - changed to tenant_name.